File: healthcare/src/main/java/com/example/zero/healthcare/dto/GoogleLoginRequest.java
package com.example.zero.healthcare.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class GoogleLoginRequest {

    @JsonProperty("idToken")
    private String idToken;

    @JsonProperty("email")
    private String email;

    @JsonProperty("nickname")
    private String nickname;

    public GoogleLoginRequest(){

    }
    public GoogleLoginRequest(String idToken,String email,String nickname){
        this.idToken = idToken;
        this.email = email;
        this.nickname = nickname;
    }
}

File: HealthCareApp/HealthCareApp/app/src/main/java/com/example/healthcareapp/data/GoogleLoginRequest.kt
package com.example.healthcareapp.data

import com.google.gson.annotations.SerializedName

data class GoogleLoginRequest(
    @SerializedName("idToken") val idToken : String,
    @SerializedName("email") val email : String,
    @SerializedName("nickname") val nickname : String




)
